ADD_NEW_VENDOR/ADD_NEW_VENDOR/Program.cs

```diff
@@ -19,6 +19,15 @@
         b => b.MigrationsAssembly("ADD_NEW_VENDOR")
     ));
 
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy("AllowAll",
+        policy => policy.AllowAnyOrigin()
```

Comment:
in allow origin allow only ui application hosted url. Since it will be the only application accessing api endpoints